com.github.camel-tooling.lsp.eclipse.updatesite/category.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-   <feature url="features/com.github.cameltooling.lsp.eclipse.client.feature_1.0.0.qualifier.jar" id="com.github.cameltooling.lsp.eclipse.client.feature" version="1.0.0.qualifier">
+   <feature url="features/com.github.cameltooling.lsp.eclipse.client.feature_1.0.0.jar" id="com.github.cameltooling.lsp.eclipse.client.feature" version="1.0.0">
```

Comment:
is that really needed to change? thought it is handled automatically.

Reply:
yes. see https://github.com/camel-tooling/camel-lsp-client-eclipse/pull/43/files#r200651887